backend/src/main/java/com/relay/service/StativService.java

```diff
@@ -0,0 +1,14 @@
+package com.relay.service;
+
+import org.springframework.stereotype.Component;
+
+import com.relay.model.statives.Stativ;
+
+@Component
+public class StativService {
+
+    public Stativ save(Stativ stativ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "stativ". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)